.ci/openshift-ci/buildall_install.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build Kata Containers and install into a given destination directory.
+#
+
+set -e
+
+export PATH="${GOPATH}/bin:$PATH"
+
+cidir="$(dirname "$0")/.."
+source /etc/os-release || source /usr/lib/os-release
+source "${cidir}/lib.sh"
+
+[ -z "$1" ] && die "Usage: $0 path/to/install/dir"
+export DESTDIR="$1"
+info "Build and install Kata Containers at ${DESTDIR}"
+
+[ "$ID" != "centos" ] && die "Expect the build root to be CentOS"
+# The scripts rely on sudo which is not installed in the build environment.
+yum install -y sudo
+"${cidir}/setup_env_centos.sh" default
+
+# The build root container has already golang installed, let's remove it
+# so that it will use the version required by kata.
+yum remove -y golang
+"${cidir}/install_go.sh" -p -f
```

Comment:
Two questions about this
1. What happens if you try to use the CentOS golang? (just curiosity on my part)
2. The install script installs go using `curl`. Should you add `yum install curl` for safety where you also install `sudo`?

Reply:
> Two questions about this
> 
>     1. What happens if you try to use the CentOS golang? (just curiosity on my part)
> 

Good question. I don't remember If I ever tried the CentOS golang. What I remember is that something used to fail if the environment had the CentOS golang package installed.

The fact is that I tried to adhere this build script as much as possible with the process used in all the Kata Jenkins jobs. And they don't use the distro's golang, instead it is installed the version suggested (`runtime/versions.yaml`)  via `tests/.ci/install_go.sh`.

>     2. The install script installs go using `curl`. Should you add `yum install curl` for safety where you also install `sudo`?

Yes, maybe. In the CentOS image used as build root container has `curl` installed so I didn't bothered.